recipes-kernel/linux/files/ae350-ax45mp/0001-Add-AE350-platform-defconfig.patch

```diff
@@ -0,0 +1,156 @@
+From ed7c9a2ee9a2ed0ab298be50fc13fdd4d0af8db1 Mon Sep 17 00:00:00 2001
```

Comment:
GitHub won't let me comment on the glibc change. Why are you changing the glibc version the bbappend applies to?

Reply:
I was going to send this PR to dunfell rather than master branch and it includes last commit on dunfell. Sorry about that.

Reply:
Ah I see. You should be able to target PRs to a specific branch.

In saying that, I think it would be better to target adding this to master. That way it will be maintained for all future releases. If you just support an old release the code won't be in master as well